app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, through: :posts
+  has_many :likes, through: :posts
```

Comment:
- [ ] With the given [_ERD_](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), whiles establishing a _has many_ **association** for _User_ model, _author_id_ (_Author_ is aliasing _User_ model) should be referenced as **foreign key** in _Comment,_ and _Like_ models. Kindly ensure to improve upon this aspect, and our code should be similar to:
```ruby
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```